src/osemosys2iamc/resultify.py

```diff
@@ -145,11 +147,7 @@ def calculate_trade(results: dict, techs: List) -> pd.DataFrame:
     years = pd.Series()
     for p in results:
         df = results[p]
-        df_f = pd.DataFrame(columns=df.columns)
-        for t in techs:
-            mask = df['TECHNOLOGY'].str.contains(t)
-            df_t = df[mask]
-            df_f = df_f.append(df_t)
+        df_f = filter_technologies(techs)
```

Comment:
The function `filter_technologies` also needs a dataframe as an argument. I guess here the `df` from the line above.

Reply:
Yes, that's correct. Line 150 should read `df_f = filter_technologies(df, techs)`